README.md

```diff
@@ -14,6 +14,12 @@ def my_lambda(event, context):
 ```
 
 ## Configuration
+* You can turn on the debug logs by setting the environment variable `LUMIGO_DEBUG=true`
+* You can change the timeout to send the trace information to the edge by setting `LUMIGO_EDGE_TIMEOUT=<seconds>`
+* You can prevent lumigo from sending keys that answer specific regexes by defining `LUMIGO_BLACKLIST_REGEX=["regex1", "regex2"]`. By default, we use the default regexes `[".*pass.*", ".*key.*"]`. All the regexes are case-insensitive.
+* In case of need, there is a kill switch, that stops all the interventions of lumigo immediately, without changing the code. Simply add an environment variable `LUMIGO_SWITCH_OFF=true`.
+* you can find examples in the examples directory 
```

Comment:
Please talk with Dori, the example directory is problematic, either fix it or remove the line. See https://github.com/lumigo-io/python_tracer/pull/87